### PR TITLE
Feature/add asking feedback in fe 165

### PR DESCRIPTION
**Pull request by TomOffringa**
This branch has a migration, so be sure to run that one first. Also, frontend doesn't run in docker anymore so if you want to test, hit me up.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "react-paginate": "^5.0.0",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
+    "react-select": "^1.1.0",
```
Comment (PM5544) [inline frontend/package.json]: This could be removed

**Review by PM5544 — CHANGES_REQUESTED** [review]
Apart from the NPM dep and the fact that I can only comment on frontend changes...
Awesome, ship it!

File: backend/flindt/api/urls.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
+from django.conf.urls import url
 from rest_framework import routers
 
 # This way of import views, does not feel good.
 from flindt.feedback import views as feedback_views
 from flindt.role import views as role_views
 from flindt.user import views as user_views
-
+from rest_framework.urlpatterns import format_suffix_patterns
```
Comment (redmerloen) [inline backend/flindt/api/urls.py]: This is not used

File: backend/flindt/api/urls.py [hunk 3]
```diff
@@ -1,10 +1,11 @@
+from django.conf.urls import url
```
Comment (redmerloen) [inline backend/flindt/api/urls.py]: This is not used

**Review by PM5544 — APPROVED** [review]

File: backend/flindt/feedback/models.py [hunk 4]
```diff
@@ -209,12 +228,39 @@ def send_feedback_skipped_message():
...
+            message = _(
+                '{} asked for feedback on the role {}. '
+                'Please be so kind and tell how you think the role is energized. {}'.format(
+                    self.recipient.get_short_name(),
+                    self.role.role.name,
+                    url,
+                )
+            )
+
+            logger.info(message)
```
Comment (redmerloen) [inline backend/flindt/feedback/models.py]: This can be removed

File: backend/flindt/feedback/models.py [hunk 5]
```diff
@@ -209,12 +228,39 @@ def send_feedback_skipped_message():
...
+
+            if not settings.DEBUG:
```
Comment (redmerloen) [inline backend/flindt/feedback/models.py]: This can also be removed

File: backend/flindt/feedback/models.py [hunk 6]
```diff
@@ -161,8 +166,22 @@ def send_feedback_received_message():
...
+                messenger = Messenger(user=self.recipient)
+                messenger.send_message(message)
+
+        def send_feedback_requested_received_message():
+            message = _(
+                'The feedback you requested is now in Flindt. Read and rate it! https://{}/#/received-feedback/{}'.
+                    format(settings.FRONTEND_HOSTNAME, self.pk)
+            )
+            logger.info(message)
```
Comment (redmerloen) [inline backend/flindt/feedback/models.py]: Can be removed now

File: backend/flindt/feedback/models.py [hunk 7]
```diff
@@ -161,8 +166,22 @@ def send_feedback_received_message():
...
+
+            if not settings.DEBUG:
```
Comment (redmerloen) [inline backend/flindt/feedback/models.py]: Can be removed

File: backend/flindt/feedback/models.py [hunk 8]
```diff
@@ -161,8 +166,22 @@ def send_feedback_received_message():
                 'You just received feedback. Read and rate it! https://{}/#/received-feedback/{}'.
                 format(settings.FRONTEND_HOSTNAME, self.pk)
             )
-            messenger = Messenger(user=self.recipient)
-            messenger.send_message(message)
+            logger.info(message)
```
Comment (redmerloen) [inline backend/flindt/feedback/models.py]: Can ben removed

File: backend/flindt/feedback/models.py [hunk 9]
```diff
@@ -161,8 +166,22 @@ def send_feedback_received_message():
...
+
+            if not settings.DEBUG:
```
Comment (redmerloen) [inline backend/flindt/feedback/models.py]: Can be removed

**Pull request merged**